dataprofiler/tests/labelers/utils.py

```diff
@@ -0,0 +1,18 @@
+import os
+import shutil
+import random
+
+import numpy as np
+
+import dataprofiler as dp
+
+
+def set_seed(seed=None):
+    """
+    Sets the see for all possible random state libraries
+    :param seed:
+    :return:
+    """
+    np.random.seed(seed)
+    random.seed(seed)
```

Comment:
Is there a reason these things aren't also set in the top level `__init__` function?

Reply:
Then these test utils just call that one? I understand this was a whole discussion previously, I am just unaware of the conclusion and the structure that was decided upon.

Reply:
We don't want to globally set the seed for other libraries.